src/hooks/use-rotate-promp-messages.ts

```diff
@@ -0,0 +1,15 @@
+import { useMemo } from 'react';
+import { useWelcomeMessages } from '../hooks/use-welcome-messages';
+
+export function useRotateWelcomeMessages( siteId: string ) {
+	const { examplePrompts } = useWelcomeMessages();
+
+	//Limit to 3 and shuffle initial prompts
+	const randomizedPrompts = useMemo( () => {
+		const shuffled = [ ...examplePrompts ].sort( () => 0.5 - Math.random() );
+		return shuffled.slice( 0, 3 );
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I disabled eslint for this line as it gives warning for siteId not being used but we want to include it to make sure prompts change for each site.

If there is a better solution, happy to apply that!